test/fixtures/neighbourhoods.yml

```diff
@@ -0,0 +1,17 @@
+# This is sufficient for like 90% of the application. We don't actually ever use region/whatever at the moment
```

Comment:
Nice

Reply:
I think any future tests {are going to/absolutely should} use fixtures so this is just a backwards compatibility thing! :)